util/string.h

```diff
@@ -24,6 +27,22 @@ constexpr bool no_case_compare(std::string_view a, std::string_view b) {
     return true;
 }
 
+inline std::string trim_start(std::string s) {
+    s.erase(begin(s), std::find_if(begin(s), end(s), [](auto ch) { return !std::isspace(ch); }));
```

Comment:
You can replace this with https://en.cppreference.com/w/cpp/string/basic_string/erase2 and not have to deal with iterators at all. Awesome new thing in C++20!

Reply:
Oh, nice. I'll fix!